cdap-app-templates/cdap-etl/cdap-data-pipeline/src/main/java/co/cask/cdap/datapipeline/SmartWorkflow.java

```diff
@@ -177,6 +190,8 @@ public void initialize(WorkflowContext context) throws Exception {
       postActions.put(actionSpec.getName(), (PostAction) context.newPluginInstance(actionSpec.getName(),
                                                                                    macroEvaluator));
     }
+
+    WRAPPERLOGGER.info("Pipeline '{}' running", context.getApplicationSpecification().getName());
```

Comment:
Better to reflect the state from the state machine than to just override with "running" string. That way any bugs/errors etc don't get masked.